### ISSUE TITLE
Fails silently if buildkite-agent isn't available

### ISSUE THREAD

**Issue by reidab** [sj26/rspec-buildkite#1]
Running with docker-compose, I'd forgotten to mount the buildkite-agent binary into the container. The `system` call returns `false` for the non-zero exit status, but it doesn't raise an exception so no message is displayed in the logs.

**Comment by TigerWolf** [comment]
I would also like to know if it is failing too. Hopefully this will help me work out what is going on.

**Comment by gondalez** [comment]
Thanks for the handy gem!

+1 for expecting a hard failure and/or error message if the agent is not present 🤔 

For others: specifying the env vars and agent mount as part of the docker-compose plugin config worked nicely for me.

`pipeline.yml`
```yml
steps:
  - label: ':rspec:'
    ...snip...
    plugins:
      - docker-compose#v3.0.3:
          ...snip...
          volumes:
            - ${BUILDKITE_AGENT_BINARY_PATH:-/usr/bin/buildkite-agent}:/usr/bin/buildkite-agent
          environment:
            - BUILDKITE
            - BUILDKITE_BUILD_ID
            - BUILDKITE_JOB_ID
            - BUILDKITE_AGENT_ACCESS_TOKEN
```

It's a bit repetitive as mentioned in https://github.com/buildkite-plugins/docker-compose-buildkite-plugin/issues/158, but hopefully that can be resolved within the buildkite plugin 🤞

**Comment by sj26** [comment]
Ooh, sorry — thanks for letting me know! I'll make sure this doesn't fail silently.

**Comment by sj26** [comment]
It should now print a warning:

<img width="811" alt="image" src="https://user-images.githubusercontent.com/14028/120403260-2e511880-c387-11eb-91f9-9f4585322197.png">

**Comment by sj26** [comment]
Released in 0.1.5

**Comment by tristanoneil** [comment]
and for anyone reading this, instead of doing what @gondalez suggests you can now just use [this option](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin/pull/285/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R362) 🎉
